webofneeds/won-owner-webapp/src/main/webapp/app/components/square-image.js

```diff
@@ -1,20 +1,48 @@
 import angular from "angular";
 
-import { generateHexColor, generateTitleCharacter } from "../utils.js";
+import { attach, generateRgbColorArray } from "../utils.js";
 
+import Identicon from "identicon.js";
+window.Identicon4dbg = Identicon;
+
+import shajs from "sha.js";
```

Comment:
the difference is 22k in bundle size, so very valid point (even if that pales hard in comparison to the rest of the bundle)

ad 1) sounds like a lot of work. i'd rather search for a smaller lib (see below)
ad 2) that might break in the future and binds the component hard to our url-structure and application. thus it's definitely the quick fix, but it seems somewhat untidy to me ^^"

smaller libs:

* https://www.npmjs.com/package/md5 (~11k)
* https://www.npmjs.com/package/md5.js (~8k)

Reply:
ok agreed -> smallest lib makes most sense

Reply:
offline discussion: instead of putting in extra work and hacking together something, we'll just stick with the sha-lib. the size difference doesn't matter much for our current bundle and the issue might solve itself (e.g. by the switch to elm or if we want to pull the crypto in the browser and we end up needing it anyway)